src/operator/tensor/ordering_op-inl.h

```diff
@@ -454,8 +457,8 @@ void TopKImpl(const RunContext &ctx,
     workspace_curr_ptr += temp_size;
   }
 
-  mxnet_op::Kernel<range_fwd, xpu>::Launch(s, batch_size * element_num, 1, 0, 1,
-    kWriteTo, indices.dptr_);
+  mxnet_op::Kernel<range_fwd, xpu>::Launch(s, batch_size * element_num, 1, static_cast<index_t>(0),
+                                           static_cast<index_t>(1), kWriteTo, indices.dptr_);
```

Comment:
Maybe just using data initializer: 
```suggestion
index_t{0}, index_t{1}
```